app/bundles/IntegrationsBundle/Auth/Provider/Oauth2ThreeLegged/Credentials/StateInterface.php

```diff
@@ -1,10 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Mautic\IntegrationsBundle\Auth\Provider\Oauth2ThreeLegged\Credentials;
-
-interface StateInterface
```

Comment:
It is tempting! But for example in this case I found that we have 3 plugins using this interface just at Acquia. There may be more from other vendors. Although I love removing code, I think we shouldn't remove classes. We should not introduce more BC breaks in the RC versions. We can mark them as deprecated and see how many people will scream. If none, we can remove them in Mautic 6.